tools/report-converter/tests/unit/sparse_output_test_files/files/sample.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+#include <sample.h>
```

Comment:
The compiler or the sparse will not find this file. You should use quotation marks to include this file:
```suggestion
#include "sample.h"
```